internal/http/services/owncloud/ocdav/get.go

```diff
@@ -109,6 +109,11 @@ func (s *svc) handleGet(w http.ResponseWriter, r *http.Request, ns string) {
 		return
 	}
 	httpReq.Header.Set(datagateway.TokenTransportHeader, token)
+
+	if r.Header.Get("Range") != "" {
```

Comment:
The `Range` header can be used with GET requests to ask for specific byte ranges in the content.

The `Content-Range` header is used to indicate which byte ranges have been returned. It follows a different format as `Range` because it may contain the full size as well.

These headers are defined in https://tools.ietf.org/html/rfc7233